packages/pods/src/Pods.ts

```diff
@@ -26,20 +26,35 @@ import {
   type TransactionRequest,
   createPublicClient,
   encodeFunctionData,
+  getAddress,
   pad,
   parseEther,
 } from 'viem'
 
 export const mint = async (
   mint: MintActionParams,
 ): Promise<TransactionFilter> => {
-  const { chainId, contractAddress, tokenId, amount, recipient } = mint
+  const { chainId, contractAddress, tokenId, amount, recipient, referral } =
+    mint
 
   const andArray1155: AndArrayItem[] = [
     {
       quantity: formatAmount(amount),
     },
   ]
+  if (referral) {
+    const checksumAddress = getAddress(referral)
+    andArray1155.push({
+      $or: [
+        { mintReferral: checksumAddress },
+        {
+          rewardsRecipients: {
+            $and: [{ $first: checksumAddress }, { $last: checksumAddress }],
```

Comment:
@Quazia We want to make sure that the referral is the only item in the array. Would using an array literal work better here? ie: `[checksumAddress]`

Reply:
Used the array literal instead of $and.